OSDOCS-11741: trouble shooting cleanup  data added to release notes (#80750)
openshift/openshift-docs: microshift_release_notes/microshift-4-17-release-notes.adoc

=== Documentation enhancements
//Doc enhancements include additions for RFEs and other continuous improvement items that are substantial and are not tied to new features or bug fixes already listed here. These can also go under the relevant section as applicable.

[id="microshift-4-17-data-cleanup_{context}"]
==== Data cleanup now documented

> With this release, various troubleshooting tasks to cleanup data are documented. For more information, see

Reviewer: ```suggestion
With this release, the `microshift-cleanup-data` script is documented. For more information, see
```

Author: done
